Domain.Sql.Tests/ScheduleCommandsDiagnosticTests.cs

```diff
@@ -0,0 +1,191 @@
+using System;
+using System.Linq;
```

Comment:
We need to put in the copyright notice at the top:

// Copyright (c) Microsoft. All rights reserved. 
// Licensed under the MIT license. See LICENSE file in the project root for full license information.